deploy/clusterrole.yaml

```diff
@@ -12,3 +12,9 @@ rules:
   - 'clusterrolebindings'
   verbs:
   - '*'
+- apiGroups:
+  - datadoghq.com
+  resources:
+  - datadogagents/finalizers
```

Comment:
I show it also in the "Role": https://github.com/DataDog/datadog-operator/pull/35/files#diff-4374306bd7fd15cc09428b7ccf9bbec8R53

Reply:
Yep we should align there